product_docs/docs/pgd/5.7/planning/limitations.mdx

```diff
@@ -76,9 +76,34 @@ and current listing.
 
 ## Mixed PGD versions
 
-PGD was developed to [enable rolling upgrades of PGD](/pgd/latest/upgrades) by allowing mixed versions of PGD to operate during the upgrade process. 
-We expect users to run mixed versions only during upgrades and, once an upgrade starts, that they complete that upgrade.
-We don't support running mixed versions of PGD except during an upgrade.
+While PGD was developed to [enable rolling upgrades of
+PGD](/pgd/latest/upgrades) by allowing mixed versions of PGD to operate during
+the upgrade process, we expect users to run mixed versions only during upgrades
+and for users to complete their upgrades as quickly as possible. We also
+recommend that you test any rolling upgrade process in a non-production
+environment before attempting it in production.
+
+When a node is upgraded, it returns to the cluster and communicates with the
+other nodes in the cluster using the lowest version of the inter-node protocol
+that is supported by all the other nodes in the cluster. This means that the
+upgraded node will be able to communicate with all other nodes in the cluster,
+but it will not be able to take advantage of any new features or improvements
+that were introduced in the newer version of PGD. 
+
+That will stay the case until all nodes in the cluster have been upgraded to the
+same newer version.  The longer you run mixed versions, the longer you will be
+without the benefits of the new version, and the longer you will be exposed to
+any potential interoperability issues that might arise from running mixed
+versions. Mixed version clusters are not supported for extended periods of time.
+
+Therefore, once an PGD cluster upgrade has begun, you should complete the whole
+cluster upgrade as quickly as possible.
+
+We don't support running mixed versions of PGD except during an upgrade, and we
+don't support clusters running mixed versions even while being upgraded, for a
```

Comment:
Proposing to remove the last "a" article on this line, as the following terms are in plural.